wdl/gwas/regenie.json

```diff
@@ -21,5 +21,8 @@
     "regenie.sub_step2.summary.docker": "eu.gcr.io/finngen-refinery-dev/saige:0.39.1-fg-pysam",
     "regenie.sub_step2.summary.finngen_annotation": "gs://r7_data/annotations/R7_annotated_variants_v0.gz",
```

Comment:
can you point the default to r8 annotation otherwise we'll for sure vetasta mopolla mettaan. :) 
e.g. currently
gs://finngen-production-library-green/finngen_R8/finngen_R8_analysis_data/annotations/R8_sisuv4_annotated_variants_v1.gz